SportsHub/SportsHub.DAL/Data/Configurations/Constants/ConfigurationConstants.cs

```diff
@@ -2,6 +2,14 @@
 {
     public static class ConfigurationConstants
     {
+        public const int SportNameMaxLength = 50;
```

Comment:
lets create separate classes inside of ConfigurationConstants for every entity

Reply:
Do you mean creating separate files containing the constants for each class and then injecting them into ConfigurationConstants or just creating the files without injecting them?

Reply:
We can have class inside of class.
 static class ConfigurationConstants{
Team{
constans
}
User{
}
...
}